src/GslibIO.jl

```diff
@@ -52,28 +92,20 @@ Optionally set the value used for missings `na`.
 """
 function load_legacy(filename::AbstractString, dims::NTuple{3,Int};
                      origin=(0.,0.,0.), spacing=(1.,1.,1.), na=-999)
-  open(filename) do fs
-    # skip header
-    readline(fs)
 
-    # number of properties
-    nvars = parse.(Int, readline(fs))
+  # parse the GSLIB file
+  X, metadata = parse_gslib(filename)                     
 
-    # properties names
-    vars = [Symbol(readline(fs)) for i in 1:nvars]
+  vars = [Symbol(v) for v in metadata.varnames]
 
-    # read property values
-    X = readdlm(fs)
+  # handle missing values
+  replace!(X, na=>NaN)
 
-    # handle missing values
-    replace!(X, na=>NaN)
+  # create data dictionary
+  data = (; zip(vars, eachcol(X))...)
+  domain = RegularGrid(dims, origin, spacing)
```

Comment:
So it seems to me that you are not using the coordinates in the point set case?

Reply:
You have missed the comment of the commit. The idea is to first have the load_legacy working with parse_legacy for a easier review. When this is done, I will submit other PR with load_legacy for point sets.

Reply:
I see. Please keep working on this PR, we don't need a separate PR. After the suggestions above are incorporated we can take a second look in the code, and generalize to the point set case.